packages/react/src/SkipLink/index.ts

```diff
@@ -0,0 +1,2 @@
+export { SkipLink } from './SkipLink'
+export type { SkipLinkProps } from './SkipLink'
```

Comment:
```suggestion
export { SkipLink, type SkipLinkProps } from './SkipLink'
```

Reply:
Discussed this, we're going with importing and exporting types on separate lines for now